subgraphs/tokemak/src/common/utils.ts

```diff
@@ -13,5 +9,5 @@ export function getTimestampInMillis(block: ethereum.Block): BigInt {
 }
 
 export function bigIntToPercentage(n: BigInt): BigDecimal {
-  return n.toBigDecimal().div(BigDecimal.fromString("100"))
+  return n.toBigDecimal().div(BigDecimal.fromString("100"));
```

Comment:
can you add `BIGDECIMAL_HUNDRED` to `constant.ts` (also in reference subgraph) and use that here?